values.yaml

```diff
@@ -41,6 +41,7 @@ server:
   # to null (the Kube cluster will pick the default).
   storage: 10Gi
   storageClass: null
+  storageClaimName: "data"
```

Comment:
Making the above-suggested changes means that this is no longer necessary.

Reply:
Hi @adilyse 
I am no longer running two installations in different namespaces. If this solves the problem than its all good. I can't really test it though. Have you?